README.md

```diff
@@ -32,29 +32,65 @@ Scikit-decide is an AI framework for Reinforcement Learning, Automated Planning
 
 The use of a virtual environment for scikit-decide is recommended, e.g. by using [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install):
 
-On MacOS/Linux:
-
-    python3 -m venv /path/to/new/virtual/environment
-    source /path/to/new/virtual/environment/bin/activate
-
-On Windows:
-
-    py -m venv \path\to\new\virtual\environment
-    \path\to\new\virtual\environment\Scripts\activate
+    conda create --name skdecide python=3.7
```

Comment:
Why moving from pyenv to conda? 
Do we need `conda`at all ?

Reply:
We do not *need* it. But 
- it was the solution proposed in the online doc => it can be changed also in online doc, no pb
- the former solution in readme was actually not selecting a version of python 
- it was perhaps easier than  
  - to install pyenv and choose an appropriate version of python
  - and to use a second tool  to create a virtual environment
- Other possibilities pipenv or poetry are more designed for developping a project (like we do for developper mode installation)

Notice that we *do* use pyenv for local install in developper mode, but it was the only solution that worked in the end for me so it seemed important to share it as is.

However, i also think it could be nice to have some steps shared between the 2 modes (recommened install vs developper install), so perhaps pyenv + venv would be better in the end (but in this case we need to change online doc also). I let you decide :)

Reply:
Ok @nhuet Let's follow your idea and share what can be shared. So remove `conda` and replace by `pyenv`. We need also to change the online doc.

Reply:
I just realized the following issue: pyenv is not for windows users. It seems that a corresponding tool [pyenv-win](https://github.com/pyenv-win/pyenv-win) has been developed for windows. I need to check on my windows laptop that commands still are working as expected though.